compiler-plugin-test/src/test/resources/test-src/plugin/package_31/sample.bal

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Fix license it should be LLC now for any new files

Reply:
Fixed in [160392e](https://github.com/ballerina-platform/module-ballerina-persist/pull/70/commits/160392ee70d4a222146d334d0e45926bd2daeeb8)